Render/RenderContext.js

```diff
@@ -256,7 +256,10 @@ class RenderContext {
         forcePaul = false,
         clearedRoomInfo = true,
         apiKey = "",
-        devInfo = false
+        devInfo = false,
+        borderColor = [0, 0, 0, 255],
```

Comment:
These new settings should also be added to the JSDoc comments at the top of this (RenderContext.js) file so they show up in tab completion